README.md

```diff
@@ -36,26 +36,20 @@ Lovelace Button card for your entities.
 - confirmation popup for sensitive items (optional)
 - support for [custom_updater](https://github.com/custom-components/custom_updater)
 
-## :warning: Long Press Support :warning:
```

Comment:
This warning is still valid. You'd have to package mwc-ripple with the card to get around this

Reply:
I'm not sure I understand why `mwc-ripple` would be linked to long press?
I deleted the reference to the import `paper-ripple`, but kept the element until I figure out how to use it without errors.
`mwc-ripple` is just for the ripple effect as far as I know.

Reply:
You're right, I believe :+1: